rule.go

```diff
@@ -448,6 +470,10 @@ func (r Rule) String() string {
 	for k, v := range r.Tags {
 		s.WriteString(fmt.Sprintf("%s:%s; ", k, v))
 	}
+	
+	for _, fb := range r.Flowbits {
```

Comment:
Please add a test case that exercises the new flowbits string functionality[0]

[0] https://github.com/google/gonids/blob/master/rule_test.go#L540